src/components/BannerComponent.vue

```diff
@@ -3,13 +3,15 @@
         <div class="row">
             <div class="col-12">
                 <div class="float-start position-absolute">
-                    <img src="../assets/images/logo_ebiodiv_small.png" class="medium-img" />
+                    <a :href="baseUrl"><img src="../assets/images/logo_ebiodiv_small.png" class="medium-img" /></a>
                 </div>
                 <h1>{{ title }}</h1>
                 <p><i>{{ subtitle }}</i></p>
             </div>
         </div>
         <div class="user-login-container">
+            <div><b-icon-question-circle></b-icon-question-circle> Help</div>
```

Comment:
add space between help and about

Reply:
done using a [`<b-nav>`](https://bootstrap-vue.org/docs/components/nav)